packages/communicator-email-ses/README.md

```diff
@@ -0,0 +1,85 @@
+# DBOS AWS Simple Email Service (SES) Communicator
+
+This is a [DBOS](https://docs.dbos.dev/) [communicator](https://docs.dbos.dev/tutorials/communicator-tutorial) for sending email using the [Amazon Web Services Simple Email Service](https://aws.amazon.com/ses/).
+
+## Getting Started
+In order to send emails with SES, it is necessary to:
+- Register with AWS and create access keys for SES. (See [Setting up SES](https://docs.aws.amazon.com/ses/latest/dg/setting-up.html) in AWS documentation.)
+- Verify a sending domain and destination addresses. (SES will initially be in "sandbox mode", which constrains email sending to be from a validated domain, to a validated email address.  See [Verified identities](https://docs.aws.amazon.com/ses/latest/dg/setting-up.html) in AWS documentation.)
+
+## Configuring a DBOS Application with AWS SES
+First, ensure that the DBOS SES communicator is installed into the application:
+```
+npm install --save @dbos-inc/communicator-email-ses
+```
+
+Second, ensure that the communicator is imported and exported from an application entrypoint source file:
+```typescript
+import { SendEmailCommunicator } from "@dbos-inc/communicator-email-ses";
+export { SendEmailCommunicator };
+```
+
+Third, place appropriate configuration into the [`dbos-config.yaml`](https://docs.dbos.dev/api-reference/configuration) file; the following example will pull the AWS information from the environment:
+```yaml
+application:
+  aws_ses_configurations: aws_config
```

Comment:
Unless we want it to be configurable in the config file alone, that was the reason for it to be there... whether the config name has to be in the code or if it can be in the config file.
Imagine everything uses 1 config, then all you do is put aws_config in the file, if you want ses to use a different config than other services, you set `aws_ses_configurations`, if you have more than one config for SES then you have no choice but specify the name in the code to distinguish them.

Reply:
I don't fully understand why we want this layer of indirection: aws_ses_configurations -> aws_config -> actual config fields.

If I want to have a specific SES config, I can write:
```typescript
const defaultSES = initClassConfiguration(SendEmailCommunicator, 'myses', {awscfgname: 'my_ses_config'});
```

In dbos-config.yaml
```
application:
   my_ses_config:
        aws_region: ${AWS_REGION}
        aws_access_key_id: ${AWS_ACCESS_KEY_ID}
        aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
```
So no need to provide another field to point to `my_ses_config`.

Reply:
I'm used to a world where people want to change the deployment configuration without changing the code... that it isn't necessarily even the same person responsible for the code and the deployment configuration.

Reply:
I see what you mean here -- you want to modify the string in `aws_ses_configurations` point to a different config. However, I think a common practice is to simply change the env variables for different deployments without modifying the config file. For example, you only need to change `${AWS_ACCESS_KEY_ID}` and `${AWS_SECRET_ACCESS_KEY}` env variables in your test vs. prod code deployment pipeline.

Reply:
But how do you change how many sets you have?  1 per service vs 1 total?

Reply:
I think the actual question was why do we need these two functions in aws-config. 
```
export function getAWSConfigs(ctx: ConfigProvider, svccfgname?: string) : AWSServiceConfig[]
export function getAWSConfigForService(ctx: ConfigProvider, svccfgname: string, cfgname: string) : AWSServiceConfig
```

Because if you need to change config or use multiple configs for a service, you'll need to update both the config file and the code. Can we simplify the logic here to simply load a block of aws-config from the dbos-config file?